### PR TITLE
Fix Dockerfile + fix release process

### LINKED ISSUE #16: Version of arlas-tagger-api is not correctly set in the release script

### LINKED ISSUE #20: The entry point in the arlas-tagger container is doubled and ends up in a shell
See the CMD and ENTRYPOINT in https://github.com/gisaia/ARLAS-tagger/blob/develop/Dockerfile-tagger#L39

Only the CMD should be kept.

### PR DESCRIPTION
**Pull request by MohamedHamouGisaia**
Fix #20 
Fix #16

### CODE REVIEW

File: docs/arlas-tagger-configuration.md [hunk 1]
```diff
@@ -75,27 +76,28 @@ docker run -ti -d \
...
+| KAFKA_CONSUMER_POLL_TIMEOUT | kafka_configuration.kafka_consumer_poll_timeout | 100 | Kafka consumer poll timeout |
+| KAFKA_BATCH_SIZE | kafka_configuration.kafka_batch_size | 10 | Kafka consumer batch size |
+| KAFKA_BROKERS | kafka_configuration.kafka_bootstrap_servers | kafka:9092 | Kafka brokers|
+| KAFKA_CONSUMER_GROUP_ID_TAGREF_LOG | kafka_configuration.kafka_consumer_group_id_tagref_log | tagref_log_consumer_group | Kafka consumer group for `tagref_log` topic |
+| KAFKA_CONSUMER_GROUP_ID_EXECUTE_TAGS | kafka_configuration.kafka_consumer_group_id_execute_tags | execute_tags_consumer_group | Kafka consumer group for `execute_tags` topic |
+| KAFKA_TOPIC_TAGREF_LOG | kafka_configuration.kafka_topic_tagref_log | tagref_log | Kafka topic for tag requests queue (tag log, retained) |
+| KAFKA_TOPIC_EXECUTE_TAGS | kafka_configuration.kafka_topic_execute_tags | execute_tags | Kafka topic for actual tag requests (actually executed, not retained) |
+| KAFKA_EXTRA_PROPS | kafka_configuration.kafka_extra_properties | - | Additional properties for configuring the kafka client. For instance `kafka_configuration.kafka_extra_properties: ssl.endpoint.identification.algorithm=https` |
```
Comment (sylvaingaudan) [inline docs/arlas-tagger-configuration.md]: `KAFKA_EXTRA_PROPS ` : the separator for the list of pairs must be documented

File: README.md [hunk 2]
```diff
@@ -45,13 +45,7 @@ Then, go to `http://localhost:9998/arlas_tagger/swagger` for testing the API.
...
 
 ```sh
-docker run arlas-tagger:latest
-```
-
-or
-
-```sh
-docker-compose -f docker-compose-tagger.yml -f docker-compose-elasticsearch.yml -f docker-compose-kafka.yml up
+docker run -d -e KAFKA_BROKERS=localhost:9092 -e ARLAS_ELASTIC_NODES=localhost:9300 -e ARLAS_ELASTIC_CLUSTER=elasticsearch  gisaia/arlas-tagger:latest
```
Comment (elouanKeryell-Even) [inline README.md]: I'm skeptical on the fact that a container can talk to the host's localhost

File: docs/arlas-tagger-configuration.md [hunk 3]
```diff
@@ -75,27 +76,28 @@ docker run -ti -d \
...
 | --- | --- | --- | --- |
-| TAGGING_STATUS_TIMEOUT | arlas-tagger.status-timeout | 3600000 | Delay before tagging status is discarded |
-| KAFKA_CONSUMER_POLL_TIMEOUT | arlas-tagger.kafka-consumer-poll-timeout | 100 | Kafka consumer poll timeout |
-| KAFKA_BATCH_SIZE | arlas-tagger.kafka-batch-size | 10 | Kafka consumer batch size |
-| KAFKA_BROKERS | arlas-tagger.kafka-bootstrap-servers | kafka:9092 | Kafka brokers|
-| KAFKA_CONSUMER_GROUP_ID_TAGREF_LOG | arlas-tagger.kafka-consumer-group-id-tagref-log | tagref_log_consumer_group | Kafka consumer group for `tagref_log` topic |
-| KAFKA_CONSUMER_GROUP_ID_EXECUTE_TAGS | arlas-tagger.kafka-consumer-group-id-execute-tags | execute_tags_consumer_group | Kafka consumer group for `execute_tags` topic |
-| KAFKA_TOPIC_TAGREF_LOG | arlas-tagger.kafka-topic-tagref-log | tagref_log | Kafka topic for tag requests queue (tag log, retained) |
-| KAFKA_TOPIC_EXECUTE_TAGS | arlas-tagger.kafka-topic-execute-tags | execute_tags | Kafka topic for actual tag requests (actually executed, not retained) |
+| TAGGING_STATUS_TIMEOUT | kafka_configuration.status_timeout | 3600000 | Delay before tagging status is discarded |
```
Comment (alainbodiguel) [inline docs/arlas-tagger-configuration.md]: It is not a kafka conf but a tagger conf

File: README.md (hunk 2 above)
Comment (MohamedHamouGisaia) [reply README.md]: It cannot, I put it as an example, I will replace it with `my-host`

File: docs/arlas-tagger-configuration.md (hunk 1 above)
Comment (MohamedHamouGisaia) [reply docs/arlas-tagger-configuration.md]: Done

File: docs/arlas-tagger-configuration.md (hunk 3 above)
Comment (MohamedHamouGisaia) [reply docs/arlas-tagger-configuration.md]: Done

File: README.md (hunk 2 above)
Comment (MohamedHamouGisaia) [reply README.md]: Done

**Pull request merged**